core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -806,10 +810,6 @@ public void start(Boolean isPrimary) throws IOException {
   @Override
   public void stop() throws IOException {
     LOG.info("Next directory id before close: {}", mDirectoryIdGenerator.peekDirectoryId());
-    if (mAsyncAuditLogWriter != null) {
-      mAsyncAuditLogWriter.stop();
-      mAsyncAuditLogWriter = null;
-    }
```

Comment:
Stop the logger in `stop()`. You may start it again in `start()`?

Reply:
It make me confusing, it remove it and never stop it for now.